libs/config/src/graph-common.config.ts

```diff
@@ -18,6 +19,10 @@ export default registerAs('graph-common', (): IGraphCommonConfig => {
       value: process.env.GRAPH_ENVIRONMENT_TYPE,
       joi: Joi.string().required().valid('Mainnet', 'TestnetPaseo'),
     },
+    atRestEncryptionKeySeed: {
+      value: process.env.AT_REST_ENCRYPTION_KEY_SEED,
+      joi: Joi.string().required(),
```

Comment:
q for future: worth adding validation that it's a valid BIP39 seed phrase? @polkadot/util-crypto has a method for this (same for provider account seed phrase, though that one would require a bit of custom validation to allow dev accounts like '//Alice' to pass)

Not important for now; just making a note.

Reply:
This seed can be anything. I don't think it has to be a BIP39 since we are using it in key derivation for X25519 keys which is used for encryption